### DISCUSSION TITLE
Alternative Mythic rewards

### DISCUSSION THREAD

**Discussion by Si1ker** [chromiecraft/chromiecraft#5915] [Suggestions]
Hello, ChromieCraft community.

As many of you may know, there has been a recent [survey](https://discord.com/channels/788063059926712341/788389203955941408/1125776125226930257) about the planned mythic rewards.
The initial idea was to provide (epic) dungeon items with an extra stat (dodge/ap/sp) on it in exchange for points that you get for completing a certain number of daily heroic (mythic) dungeons.

> In conclusion, if you do your daily Mythic, you will be able to choose 1 item every 5 days.
> 
> However, of course, the bravest of the brave deserve better than just another item. All items sent by Chromie have an additional permanent enchant, which is not affected by other regular enchants or gems. You can freely add gems and regular enchants to your mythic item without impacting this bonus enchant.
source: https://www.chromiecraft.com/en/mythic-dungeons-raids-release-schedule/

The survey [result](https://www.chromiecraft.com/en/mythic-dungeons-raids-release-schedule/) shows the majority (by a short) of the community is against slightly enhanced epic gear which would have been a great compensation for the increased [difficulty](https://www.chromiecraft.com/en/mythic-mechanics-in-tbc-heroics/).

As per the nature of suggestions in this repository, I'd like to present an alternative **idea** for the mythic system to avoid mythic dungeons' death on arrival.

1. Combine mythic raid points and mythic dungeon points into one currency (referred to as "mythic points" from this point on.)
2. Daily mythic dungeon: 1 mythic point  //  Mythic raid boss: 3 mythic points
(The amount of points proposed is **just an example**, values can be lower/higher as the Staff may seem correct.)
3. List raid items for a (high) price of XX mythic points (or have prices by category depending on item slot).
4. Either remove heroic dungeon epic items or keep them for sale at a lower price.

* Obtaining raid items through harder heroic dungeons has been introduced in Wotlk Classic to keep the dungeons attractive:
> Heroic Plus Dungeons
> When Ulduar releases in Phase 2 of Wrath Classic, all 10-man loot from Phase 1 of Wrath Classic will be moved to Heroic Dungeons, in a new version of the dungeon with increased difficulty. Players will have the ability to increase the difficulty on Heroic Dungeons, granting the loot drops from 10-man Phase 1 raids.
source: https://www.wowhead.com/wotlk/news/loot-changes-and-heroic-plus-dungeons-coming-in-wrath-classic-328956

### Final notes:
- Mythic dungeon [extra mechanics](https://www.chromiecraft.com/en/mythic-mechanics-in-tbc-heroics/) seem very hard and will present a real challenge, so this is not "free loot" as many may think. 
- The amount of points proposed is just an example, values can be lower/higher as the Staff may seem correct.
- The ultimate goal of this suggestion is to make the 48.3% of the community that voted "yes" happy while keeping the 51.7% of pop that doesn't want "custom gear", plus avoid all time/effort/excitement of the Staff going to waste. = Everyone wins.

**Comment by Never6** [comment]
Mythic doesn't need rewards, the reward is not having your time feel wasted by doing not faceroll content.

**Reply by elthehablo** [reply]
If you think for a second that without rewards mythic is going to not be dropped after the first attempt because the normal mode is way easier to farm then I do not know what to say. Troll take

**Reply by Never6** [reply]
Then why have an option at all?
Literally nobody minded vanilla raids, you decided to tune raids BECAUSE fights would be <1min for the average guild
Guess what, gruul and mag  and world bosses released with little to no tuning and how long are fights? 
You had a good system working with buffed raids (that didn't get bonus rewards) What happend to if it ain't broke don't fix it. Why was it even changed? Cause people who didn't wanna farm NR gear couldn't get past huhuran.
That's pathetic

**Reply by Si1ker** [reply]
I support 100% the old (AQ40-like) tunning and the "if it ain't broke don't fix it".

(IMHO) If the survey could have had a third option "bring back old vanilla tunning", that would have been the winner.

**Reply by shaszun** [reply]
So far I haven't heard a person to say they didn't like AQ40 even when there are still not fully fixed bosses. The idea of Mythics is good, but this is the case with a true TBC server. In wrath it is not possible, you already have the better talents and gear and if you include the additional stats from Mythic gear. Faceroll will be a strong word to say in that case. Also a lot of people are not fond with custom mechanics, blizzard already have mechanics balanced 15 years ago, now they need tunning for our current stats and jobs is done.

**Comment by balleny** [comment]
something i am missing in this proposal is how future content is handled. lets say you have 100 tokens and t5 becomes available including t5-mythic.
how should/could that get handled.

**Reply by Si1ker** [reply]
I'd say to wipe the tokens/points on new releases like Wotlk does with emblems so it is a fresh start to grind (keep alive pop) again.

**Reply by balleny** [reply]
but wouldnt this sound a bit unfair to the people that dont care about raiding but focus on 5 player dungeons mostly?

**Reply by Si1ker** [reply]
I think there's no "unfair" here since they don't even have a way to get raid items if they don't care about raiding in the first place (current state).
